service/src/test/java/uk/gov/hmcts/reform/fpl/service/HearingBookingServiceTest.java

```diff
@@ -105,14 +105,14 @@ void shouldGetMostUrgentHearingBookingFromACollectionOfHearingBookings() {
     @Test
     void shouldGetHearingBookingWhenKeyMatchesHearingBookingElementUUID() {
         List<Element<HearingBooking>> hearingBookings = createHearingBookings();
-        HearingBooking hearingBooking = service.getHearingBookingByUUID(hearingBookings, HEARING_IDS[2]);
+        HearingBooking hearingBooking = HearingBookingService.getHearingBookingByUUID(hearingBookings, HEARING_IDS[2]);
```

Comment:
probably it would be better to make it as not static method in service

Reply:
tbh the method should probs just be deleted, we have the `ElementUtils.findElement(...)` that does the same work in a generic case and returns an optional which is probs better than null